docroot/themes/custom/su_humsci_theme/su_humsci_theme.libraries.yml

```diff
@@ -92,6 +92,7 @@ base:
   version: VERSION
   css:
     base:
+      css/base/archaeology.css: {}
```

Comment:
Lets move this to its own library so that its easier to turn on/off or move to another theme later.